integration_error.py

```diff
@@ -0,0 +1,2 @@
+class IntegrationError(Exception):
```

Comment:
Why in a separate file?
Let's move it to start_integration.py

Reply:
I think that in the future IntegrationError will be needed in sms_notifservice.py. It seems to me that it would be good to fix exception handling in the current integration

Reply:
Ok. Then move it to sms_notifservice.py. And rename it to ModuleError

Reply:
I'm not sure if this class goes well with sms_notifservice. And I'm not sure what we need to rename now, we will end up with a mess of different names.
I propose to fix the exception handling in a separate Issue, then it will be clear what to do with this class. And in a separate Issue, rename integration to module.

Reply:
Ok